Formula/bash-git-prompt.rb

```diff
@@ -17,11 +17,15 @@ def install
   end
 
   def caveats; <<~EOS
-    You should add the following to your .bashrc (or equivalent):
+    You should add the following to your .bashrc (or .bash_profile if bash is your login shell):
       if [ -f "#{HOMEBREW_PREFIX}/opt/bash-git-prompt/share/gitprompt.sh" ]; then
         __GIT_PROMPT_DIR="#{HOMEBREW_PREFIX}/opt/bash-git-prompt/share"
         source "#{HOMEBREW_PREFIX}/opt/bash-git-prompt/share/gitprompt.sh"
       fi
     EOS
   end
+
+  test do
```

Comment:
We can have a real test. See for example how `liquidprompt` handles it.

Reply:
@fxcoudert Fixed ;)